Add Cloud Events support for #55 (#56)
GoogleCloudPlatform/functions-framework-python: examples/cloudevents/README.md

> # Python Functions Frameworks Cloud Event Sample
> This sample uses the [Cloud Events SDK](https://github.com/cloudevents/sdk-python) to receive an event.
> ## How to run this locally
> Build the docker image.

Reviewer: ```suggestion
# Deploying a CloudEvent function to Cloud Run with the Functions Framework
This sample uses the [Cloud Events SDK](https://github.com/cloudevents/sdk-python) to send and receive a CloudEvent on Cloud Run.

## How to run this locally
Build the Docker image:
```

```commandline
docker build --tag ff_example .
```

> Run the image and bind the correct ports.

Reviewer: ```suggestion
Run the image and bind the correct ports:
```

```commandline
docker run -p:8080:8080 ff_example
```

> Send an event to the container.

Reviewer: ```suggestion
Send an event to the container:
```
